crfs.go

```diff
@@ -1009,6 +1009,46 @@ func (n *node) Readlink(ctx context.Context, req *fuse.ReadlinkRequest) (string,
 	return n.te.LinkName, nil
 }
 
+// Listxattr lists the extended attributes specified for the node.
+//
+// See https://godoc.org/bazil.org/fuse/fs#NodeListxattrer
+func (n *node) Listxattr(ctx context.Context, req *fuse.ListxattrRequest, resp *fuse.ListxattrResponse) error {
+	var allXattrs []byte
+	for k, _ := range n.te.Xattrs {
+		allXattrs = append(append(allXattrs, []byte(k)...), 0)
+	}
+
+	if req.Position >= uint32(len(allXattrs)) {
+		resp.Xattr = []byte{}
+		return nil
+	}
+	allXattrs = allXattrs[req.Position:]
+	if req.Size > 0 && req.Size < uint32(len(allXattrs)) {
+		return syscall.ERANGE
```

Comment:
Returning syscall.ERANGE results in EIO because [bazil/fuse's errno type](https://github.com/bazil/fuse/blob/65cc252bf6691cb3c7014bcb2c8dc29de91e3a7e/fuse.go#L321) isn't compatible with `syscall.Errno` and fallbacks into default errno(EIO). I think returning EIO is confusing for the filesystem user.

Simply, by deleting this check all work fine on my laptop [because bazil/fuse do this check in a proper way](https://github.com/bazil/fuse/blob/65cc252bf6691cb3c7014bcb2c8dc29de91e3a7e/fs/serve.go#L1146).